cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -1201,32 +1202,40 @@ instance Condense (HeaderHash blk)
                  , "tip" .= (String (pack $ showTip verb tip))
                  , "rolledBackBlock" .= (String (pack $ showPoint verb pt)) ]
 
-instance (Show (GenTx blk), Show (GenTxId blk))
+instance (Condense (GenTx blk), Condense (GenTxId blk), Show (ApplyTxErr blk))
  => ToObject (TraceEventMempool blk) where
-  toObject _verb (TraceMempoolAddedTx tx _mpSzBefore mpSzAfter) =
+  toObject verb (TraceMempoolAddedTx tx _mpSzBefore mpSzAfter) =
     mkObject
       [ "kind" .= String "TraceMempoolAddedTx"
-      , "txAdded" .= String (pack $ show tx)
-      , "mempoolSize" .= String (pack $ show mpSzAfter)
+      , "tx" .= condense tx
```

Comment:
Should we avoid using `condense` as per https://github.com/input-output-hk/cardano-node/issues/721#issuecomment-604343274

Reply:
yes, condense should be banned.

Reply:
Ok, then we should use `ToObject` and provide an instance for it, and add it to `TraceConstraints`.